### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by chemik-bit**
Added docker functionality.

### CODE REVIEW

File: Dockerfile.backend
[the remark this answers is not in the document]
Comment (filip-michalsky) [reply Dockerfile.backend:1]: @iljamak unfortunately, Docker does not play nicely with poetry, right @chemik-bit ? So i would probably keep as is for the docker deployment.

File: frontend/Dockerfile.frontend
[the remark this answers is not in the document]
Comment (filip-michalsky) [reply frontend/Dockerfile.frontend:1]: @iljamak I think this is fine for now - don't see an issue with it? Please add as an issue into "Issues" for a future consideration

File: README.md [hunk 1]
```diff
@@ -213,15 +213,32 @@ For more detailed installation steps along with the reasons for doing each pleas
...
+
+   `git clone https://github.com/filip-michalsky/SalesGPT.git`
+
+2. Navigate to the SalesGPT directory:
+
+   `cd SalesGPT`
+
+3. Run the application using Docker Compose:
+
+   `docker-compose up`
```
Comment (filip-michalsky) [inline README.md:236]: `docker compose up -d --build` ?

if errors, run:
`docker system prune --volumes`
`docker compose up -d --build --no-cache` ?

File: README.md [hunk 2]
```diff
@@ -213,15 +213,32 @@ For more detailed installation steps along with the reasons for doing each pleas
...
+
+This will build and start the necessary Docker containers for SalesGPT. You can then access the application by opening the following link in any of your browsers: [localhost:3000/chat](http://localhost:3000/chat)
```
Comment (filip-michalsky) [inline README.md:238]: let's route the root to redirect to /chat

**Review by filip-michalsky — CHANGES_REQUESTED** [review]
great job! requested a few changes including keeping some instructions which were deleted.

File: Dockerfile.backend
[the remark this answers is not in the document]
Comment (chemik-bit) [reply Dockerfile.backend:1]: Docker does not currently work with poetry too well, so pip is commonly used. If poetry support is extended will transfer right away!

File: frontend/Dockerfile.frontend
[the remark this answers is not in the document]
Comment (chemik-bit) [reply frontend/Dockerfile.frontend:1]: I added this as a new issue. Will address in the future, will increase priority if upvoted by users.

**Comment by filip-michalsky** [comment]
@chemik-bit great job!

**Review by filip-michalsky — APPROVED** [review]

**Pull request merged**